.cirrus.bazelrc

```diff
@@ -0,0 +1,6 @@
+# https://docs.bazel.build/versions/master/remote-caching.html
```

Comment:
Are you planning on setting up official CI for this project? If so, cool!

If this is only intended for you to test changes locally, please remove this. The settings provided here are specific to the environment at hand. At Prodrive, for example, we would use remote execution instead of sandboxed local execution.

Reply:
Cirrus-CI provides caching for free :-)
And yes, I have a hard time to have a local dev branch with Cirrus and send a pull request that only contains what I've added on top of that. I'll clean this PR, of course.

Reply:
Reverted